modules/ipi-verifying-nodes-after-installation.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// *
+
+:_module-type: PROCEDURE
+[id="ipi-verifying-nodes-after-installation_{context}"]
+= Verifying node state after installation
+
+[role="_abstract"]
+//Short introductory paragraph that provides an overview of the module. The text that immediately follows the abstract tag is used for search metadata.
+The {product-title} installation completes when the following installation health checks are successful:
+
+* The provisioning host can access the OpenShift web console.
+
+* All master nodes are ready.
+
+* All cluster Operators are available.
+
+[NOTE]
+====
+It can take some time after cluster installation is successfully completed before all worker nodes report as `READY`. The Cluster Operator continuously attempts to provision worker nodes for a set period of time. The state of a worker node only changes to `DEGRADED` when the Cluster Operator fails to provision the worker node within this time period. You can define this time period in the `install-config.yaml` file.
+==== 
```

Comment:
Q: How can you configure a time-out value? Or is there a default time-out value that we should just state here?

Reply:
Well this is my Q for you. No idea tbh ;)

Reply:
I updated the note. What do you think Alfonso?

Reply:
Maybe we should make explicit that the Degraded state for _Machine Api Operator_ and _Baremetal Operator_ depends only on the operators own resources and not the state of the nodes ?

Reply:
Updated the note.

Reply:
Got a lgtm from Alfonso: https://bugzilla.redhat.com/show_bug.cgi?id=2084113#c8